tutorials/week1-installing-postman.md

```diff
@@ -0,0 +1,186 @@
+# Tutorial: How to Install and Use Postman
```

Comment:
We should update the title to reflect the new content. Do we need to update the file name to follow any convention for the course @abhutta0?

Reply:
File name can stay as Week1. We should update the title

Reply:
Renamed it to: tutorials/week1-installing-postman.md